packages/json/src/parser/array.ts

```diff
@@ -1,11 +1,11 @@
 import * as core from '@spyglassmc/core'
 import type { JsonArrayNode } from '../node/index.js'
-import { entry } from './entry.js'
+import { json } from './entry.js'
 
 export const array: core.Parser<JsonArrayNode> = (src, ctx) => {
 	const parser = core.list({
 		start: '[',
-		value: entry,
+		value: json(),
```

Comment:
Afaik this will create a new function every time the `array` parser is used to parse something. Probably not a big deal but might be better to store a constant like `entry` in `./entry.ts` with `json(false)`.

Reply:
done in [`5bffcff` (#1132)](https://github.com/SpyglassMC/Spyglass/pull/1132/commits/5bffcffc5f719f60516d5b07be139491b97a9ea7)